src/index.html

```diff
@@ -15,8 +15,109 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <a
+        class="logo"
+        href="index.html"
+      >
+        <img
+          class="logo__img"
```

Comment:
You used a good organization both here and in the nav (I'll mark with a *) but you should implement mix block or just use normal nesting (which I don't recommend because you would need to refactor it)

https://en.bem.info/methodology/quick-start/#nesting-1
Mix Block: https://mate.academy/en/learn/css-advanced/bem?section=video&videoId=4936